### PR TITLE
fix: UX improvement to form dialogs and updated library

### LINKED ISSUE #5138: Minor: Add tooltip documentation for properties of type 'list'
## Is your feature request related to a problem? Please describe.
Minor issue when entering multiple values for a property with property type list. Intuitively I separate list values by a comma which leads to the red error border to appear around the property. Nothing in the UI indicates why the comma seperated list is erroneous or tell the user to add each item and click 'enter' to indicate a new entry as opposed to a comma.

## Describe the solution you'd like

It makes sense to not use a comma as a separator as an individual list item may want to contain a comma, so perhaps adding to the tool tip for Accepted Values some documentation on how to separate list items or a more specific error when the red border is rendered beyond 'property has error(s)'

## Additional context

image

### LINKED ISSUE #5141: Need more guidance on creating a form dialog
Screenshot 2020-12-04 at 13 30 40

### LINKED ISSUE #5143: Status message needed after "generate dialog" is done
Screenshot 2020-12-04 at 12 50 02

### LINKED ISSUE #5153: [R11 Bugbash] Where is the export schema?
## Describe the bug

There is import schema on the form dialog UI, but where is the export so I can import?

## Version

                Bot Framework Composer

                Version:  1.2.1-nightly.191674.827857c
                Electron: 8.2.4
                Chrome: 80.0.3987.165
                NodeJS: 12.13.0
                V8: 8.0.426.27-electron.0

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Go to "Forms (Preview)" UI
2. Click "+" button to create new form dialog
3. Give it a name
4. Click "Import schema"

## Expected behavior

Able to find something to import. If the form is already there for my project, why do I need to import it? If I am sharing it with someone then how do I export and share it?

### PR DESCRIPTION
**Pull request by hatpick**
## Description

1. Adds notification to form dialog generation: progress, fail, success.
2. Adds help text to schemas, fixes minor search issue for schemas.
3. Adds back export for form dialog schema.
4. Replace native browser "confirm" with Composer confirm dialog (moved to ui-shared)
    Note: using native browser confirm freezes electron UI on windows.
5. Fixes disable state for "clear all" and "generate dialog"
6. Fixes text for feature flags view and in the documentation.

## Task Item

closes #5153
closes #5143
closes #5141
closes #5138

### CODE REVIEW

File: Composer/packages/client/src/pages/form-dialog/FormDialogPage.tsx [hunk 1]
```diff
@@ -96,6 +112,51 @@ const FormDialogPage: React.FC<Props> = React.memo((props: Props) => {
     [navigateToGeneratedDialog, projectId]
```
Comment (GeoffCoxMSFT) [inline Composer/packages/client/src/pages/form-dialog/FormDialogPage.tsx:112]: This uses schemaId, does it need to be in the list of dependencies?

File: Composer/packages/client/src/recoilModel/atoms/appState.ts [hunk 2]
```diff
@@ -235,6 +235,11 @@ export const formDialogGenerationProgressingState = atom({
   default: false,
 });
 
+export const formDialogErrorState = atom<(Error & { kind: 'templateFetch' | 'generation' | 'deletion' }) | undefined>({
```
Comment (GeoffCoxMSFT) [inline Composer/packages/client/src/recoilModel/atoms/appState.ts:238]: Nit: I thought you discouraged these anonymous param types ;)

File: Composer/packages/client/src/recoilModel/dispatchers/formDialogs.ts [hunk 3]
```diff
@@ -55,17 +57,20 @@ export const formDialogsDispatcher = () => {
...
 
       set(formDialogLibraryTemplatesState, templates);
-    } catch (error) {
-      set(applicationErrorState, {
-        message: error.message,
-        summary: formatMessage('Load form dialog schema templates Error'),
+    } catch (ex) {
+      set(formDialogErrorState, {
+        ...ex,
+        message: formatMessage('Load form dialog schema templates Error'),
```
Comment (GeoffCoxMSFT) [inline Composer/packages/client/src/recoilModel/dispatchers/formDialogs.ts:63]: Nit: This reads strangely Error

File: Composer/packages/form-dialogs/src/components/FormDialogPropertiesEditor.tsx [hunk 4]
```diff
@@ -134,15 +147,31 @@ export const FormDialogPropertiesEditor = React.memo((props: Props) => {
       key: 'import',
       onRender: () => <CommandBarUploadButton accept={schemaExtension} disabled={isGenerating} onUpload={upload} />,
     },
+    {
+      key: 'export',
+      iconProps: { iconName: 'Export' },
```
Comment (GeoffCoxMSFT) [inline Composer/packages/form-dialogs/src/components/FormDialogPropertiesEditor.tsx:152]: Super nit: could be a const object for iconProps

File: Composer/packages/lib/ui-shared/src/components/ConfirmDialog.tsx [hunk 5]
```diff
@@ -9,10 +9,12 @@ import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button'
 import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
 import ReactDOM from 'react-dom';
 import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
-import { SharedColors } from '@uifabric/fluent-theme';
 import formatMessage from 'format-message';
 
-import { dialogStyle } from './dialogStyle';
+const dialogStyle = {
```
Comment (GeoffCoxMSFT) [inline Composer/packages/lib/ui-shared/src/components/ConfirmDialog.tsx:14]: YAY!

**Review by GeoffCoxMSFT — APPROVED** [review]

File: Composer/packages/client/src/recoilModel/atoms/appState.ts (hunk 2 above)
Comment (hatpick) [reply Composer/packages/client/src/recoilModel/atoms/appState.ts:238]: the union type doesn't need import (++ for union type vs. enum) so this inline type never needs to be imported anywhere.

**Review by GeoffCoxMSFT — APPROVED** [review]

**Pull request merged**